plugins/integrations/kubernetes-service/src/main/java/com/cloud/kubernetes/cluster/KubernetesClusterService.java

```diff
@@ -38,7 +38,7 @@ public interface KubernetesClusterService extends PluggableService, Configurable
 
     static final ConfigKey<Boolean> KubernetesServiceEnabled = new ConfigKey<Boolean>("Advanced", Boolean.class,
             "cloud.kubernetes.service.enabled",
-            "false",
+            "true",
             "Indicates whether Kubernetes Service plugin is enabled or not. Management server restart needed on change",
             false);
```

Comment:
```suggestion
            true);
```

Reply:
It seems it’s not dynamic config

Reply:
I have not found a reason why it can't be (yet)